server/msgapi.py

```
@@ -109,6 +111,75 @@ def process_markdown(text):
     
     return image_links, modified_text, split_text
 
+# 富文本适配
+def process_rich_text(delta):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)